pycqed/instrument_drivers/physical_instruments/ZurichInstruments/UHFQuantumController.py

```diff
@@ -383,25 +333,6 @@ def _check_versions(self):
     def _num_channels(self):
         return 2
 
-    def _reset_awg_program_features(self):
```

Comment:
It appears that the _reset_awg_program_features method was removed after the code refactoring. I do not know if this was intended or not, or even whether we are currently making use of the functionality implemented by this method